### PR TITLE
(ft-commitment) naming convention corrected

### PR DESCRIPTION
**Pull request by LijuJoseJJ**
# Description
This PR correct naming convention for `ft-commitment` across all microservices, also changes database schema for `ft-commitment`

## How Has This Been Tested
1. By running UI
2. By running Integration test (```./nightfall-test```)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by LijuJoseJJ** [comment]
please do the test this PR by running UI

File: zkp/src/f-token-controller.js [hunk 1]
```diff
@@ -403,13 +403,13 @@ async function transfer(
     account,
     fTokenShieldInstance,
     inputCommitments[0].commitment,
-    inputCommitments[0].index,
+    inputCommitments[0].commitment_index,
```
Comment (willKim19) [inline zkp/src/f-token-controller.js]: I think these should be camelCased

File: database/src/business/ft-commitment.service.js [hunk 2]
```diff
@@ -16,42 +16,32 @@ export default class FtCommitmentService {
...
-    if (isTransferred) mappedData = ftCommitmentTransferTransactionMapper(data);
-    else if (isBatchTransferred) mappedData = ftCommitmentTransferTransactionMapper(data);
-    else mappedData = ftCommitmentMapper(data);
+    const mappedData = ftCommitmentTransferTransactionMapper(data);
 
     if (isReceived)
       return this.ftCommitmentTransactionService.insertTransaction({
         ...mappedData,
-        type: 'received',
+        transaction_type: 'transfer_incoming',
```
Comment (willKim19) [inline database/src/business/ft-commitment.service.js]: camelCase

File: database/src/mappers/ft-commitment.js [hunk 3]
```diff
@@ -1,76 +1,38 @@
...
+  const [{ value, salt, commitment, commitmentIndex, owner }] = outputCommitments;
+  return {
+    [value && 'value']: value,
+    [salt && 'salt']: salt,
+    [commitment && 'commitment']: commitment,
+    [commitmentIndex !== undefined ? 'commitment_index' : undefined]: commitmentIndex,
+    owner: { ...owner, public_key: owner.publicKey },
 
-    [zCorrect || zCorrect === false ? 'coin_commitment_reconciles' : undefined]: zCorrect,
+    [zCorrect || zCorrect === false ? 'commitment_reconciles' : undefined]: zCorrect,
```
Comment (willKim19) [inline database/src/mappers/ft-commitment.js]: camelCase

File: database/src/models/ft-commitment-transaction.model.js [hunk 4]
```diff
@@ -2,61 +2,88 @@ import { Schema } from 'mongoose';
 
 export default new Schema(
   {
-    type: {
+    transaction_type: {
```
Comment (willKim19) [inline database/src/models/ft-commitment-transaction.model.js]: camelCase

File: zkp/src/f-token-controller.js (hunk 1 above)
Comment (LijuJoseJJ) [reply zkp/src/f-token-controller.js]: changes done

File: database/src/business/ft-commitment.service.js (hunk 2 above)
Comment (LijuJoseJJ) [reply database/src/business/ft-commitment.service.js]: changes done

File: database/src/models/ft-commitment-transaction.model.js (hunk 4 above)
Comment (LijuJoseJJ) [reply database/src/models/ft-commitment-transaction.model.js]: changes done

File: database/src/mappers/ft-commitment.js (hunk 3 above)
Comment (LijuJoseJJ) [reply database/src/mappers/ft-commitment.js]: changes done

**Pull request merged**